server/ServerApp.cpp

```diff
@@ -1747,16 +1752,18 @@ void ServerApp::DropPlayerEmpireLink(int player_id)
 { m_player_empire_ids.erase(player_id); }
 
 int ServerApp::AddPlayerIntoGame(const PlayerConnectionPtr& player_connection) {
+    Empire* empire_ptr = nullptr;
```

Comment:
prefer just `empire`. that it is a pointer is apparently clear enough from the use of `->` to access the member functions.

Reply:
Fixed.